pkg/launch/launchable.go

```diff
@@ -17,9 +17,13 @@ type LaunchableID string
 
 func (l LaunchableID) String() string { return string(l) }
 
+type LaunchableVersionID string
```

Comment:
I think it's fine to make version strings a distinct type. It seems wordy to call this a "version _identifier_". Typically, wouldn't developers just refer to the "version" of their code?